src/Controller/AddOnOptionsController.py

```diff
@@ -398,12 +398,12 @@ def accepting(self):
     #                                                                                                                 #
     ###################################################################################################################
 
-    def fillTables(self):
+    def fill_tables(self):
         # Fill the Windowing table
-        with open(resource_path("data/csv/imageWindowing.csv"), "r") as fileInput:
-            next(fileInput)
+        with open(resource_path("data/csv/imageWindowing.csv"), "r") as file_Input:
```

Comment:
if refactoring with renaming to meet python standards/defaults...
I believe that snake case for variables is appropriate,
file_input
rather than snake plus Camel Case...

Reply:
snake_case is correct, unfortunately I missed the capital "I". Thank you for pointing that out.

Reply:
Please update with a commit or create a separate issue for follow up.
Prefer an update / commit because this code is exactly what was touched.

Reply:
I've just pushed a commit up to fix this, thank you!